Migration script/openIMIS migration latest.sql

```diff
@@ -182,6 +182,19 @@ BEGIN
 END
 GO
 
+IF OBJECT_ID('tblBulkControlNumbers') IS NULL
+BEGIN
+	CREATE TABLE tblBulkControlNumbers 
+	(
```

Comment:
it is not aligned with the ONLINE init script

Reply:
what do you mean? Can you please refer to the last commit?